enclave/crypto/cert.c

```diff
@@ -96,15 +96,18 @@ OE_INLINE void _referent_add_ref(Referent* referent)
 OE_INLINE void _referent_free(Referent* referent)
 {
     /* If this was the last reference, release the object */
-    if (oe_atomic_decrement(&referent->refs) == 0)
+    if (referent)
```

Comment:
For the code readability, consider do the check and early return instead of introducing nested brackets.
For example, `if (!referent) return;`

Reply:
Got it. I will change it now.